### PR TITLE
Update deadpool.ms

### PR DESCRIPTION
**Pull request by MrCraft888**

### CODE REVIEW

File: CommandHelper/LocalPackages/commands/buffs/deadpool.ms [hunk 1]
```diff
@@ -14,6 +20,7 @@
                 "/jump 0-4",
                 "/run 0-3",
                 "/haste 0-2"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/deadpool.ms]: missing a comma

File: CommandHelper/LocalPackages/commands/buffs/deadpool.ms [hunk 2]
```diff
@@ -14,6 +20,7 @@
...
+                "/jumpto"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/deadpool.ms]: Requires proof

File: CommandHelper/LocalPackages/commands/buffs/deadpool.ms [hunk 3]
```diff
@@ -6,6 +6,12 @@
...
+                array(  
+                    'sig': 'uc6ajWnofBWExjF1axV5OXO9QUKYvCO0GtPAsYVFnK/APiEWJh/ub3cnLfx55YpkTr4VaBoTUJVpUmwveEDH1fS7Py6Jnf2o4Sh+3rCDKKkDWnZS3s2kns0+bIINgGEJ9lI9s8F3NnljoKhzG0F74xtMon6taJB6cmVrAzN77rbm5l/Sra3lxmvz2oanmVxiooNmFjVkviRXXsCP19+aj/jzvdhxJLc4srCua/vrDduk3gE/uk60A4qotiehSVNiG0hSZdAps+Zy5IStVU7ijlU75oYj40siPAadXA+AmN8V6k+iIDhDJQJAhgEmPH/08ascOcROLSAHF6wHzKmZdaty/xsgm+en6lsB9VVnWINWu0yyzfO3rLlH/GgbkA3m4tuBlvBSbQUVELmVZ7KbTFjxF8gJKhaNdUtWMNu9S2NKHNOyj233glq4bQEt4ihwl/ofLytmZr0NOhBab/onQck/dq30Kf+Ta11MTFF2+jv9CAZOFiDyTYjTpC6Tp08nYupMKH9IKr9pehrmNdsUHbb7K8m1dwhG3pY5JTLSAcDitOGHYNfdA51YeZoBclTf/twEIE0aOLe4crjnQa1Scys6qBkf2KZYjzFJFgWx5nPSFOrn2lBv1LR2XJdwbZl9brQ/HdOGj3s8hV4lJjOZGRsTHToTHQbyBHLIPl5UrK0=',                    
+                    'val': 'eyJ0aW1lc3RhbXAiOjE0ODc5OTM1NTQ0NzYsInByb2ZpbGVJZCI6IjQ3NDcwN2Y3ZjhiNDQzZWE4OTc4OWNhZTE3OTE3ZTgyIiwicHJvZmlsZU5hbWUiOiJNckNyYWZ0ODg4Iiwic2lnbmF0dXJlUmVxdWlyZWQiOnRydWUsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS9mMmIyNGUyYmUzMDQwYmFiOWNmY2M2ZmJkOTJhYzQ5ZDVjN2Y0YjY2NmQ4M2FlMmVlNmMyN2NlNzNjYTNjYyJ9fX0='
+                )
+            ),
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/deadpool.ms]: requires a link to review the skin

File: CommandHelper/LocalPackages/commands/buffs/deadpool.ms (hunk 2 above)
Comment (MrCraft888) [inline CommandHelper/LocalPackages/commands/buffs/deadpool.ms]: https://www.youtube.com/watch?v=XVsyFloxdsc

Comment (MrCraft888) [inline CommandHelper/LocalPackages/commands/buffs/deadpool.ms]: thats not a fan made game its a Marvel Game

File: CommandHelper/LocalPackages/commands/buffs/deadpool.ms (hunk 3 above)
Comment (MrCraft888) [inline CommandHelper/LocalPackages/commands/buffs/deadpool.ms]: http://www.planetminecraft.com/skin/deadpoolx-men-origins/ i couldnt find  anything better

File: CommandHelper/LocalPackages/commands/buffs/deadpool.ms (hunk 1 above)
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/deadpool.ms]: missing a comma

**Pull request closed without merging**